src/pretix/plugins/banktransfer/payment.py

```diff
@@ -118,6 +118,16 @@ def test_mode_message(self):
     def settings_form_fields(self):
         d = OrderedDict(
             list(super().settings_form_fields.items()) + list(BankTransfer.form_fields().items()) + [
+                ('code_prefix', forms.CharField(
+                    label=_('Bank transfer code prefix.'),
```

Comment:
I think the context of bank transfer is clear, let's jsut call it "Reference code prefix" (without a fullstop at the end).